scripts/ingests/utils.py

```diff
@@ -48,6 +52,26 @@ def verboseprint(*args, **kwargs):
 #     sys.tracebacklimit = default_value  # revert changes
 
 
+def load_simpledb(RECREATE_DB=True):
+    # Utility function to load the database
+
+    db_file = 'SIMPLE.db'
```

Comment:
I feel like `db_file` should be a parameter that the user can pass to the `load_simpledb` function, even it has a default value of `'SIMPLE.db'`

Reply:
agreed.